src/Dfe.PlanTech.Application/Content/Queries/GetRecommendationIntroQuery.cs

```diff
@@ -0,0 +1,13 @@
+using Dfe.PlanTech.Domain.Content.Queries;
+using Dfe.PlanTech.Domain.Questionnaire.Enums;
+using Dfe.PlanTech.Domain.Questionnaire.Models;
+
+namespace Dfe.PlanTech.Application.Content.Queries;
+
+public class GetRecommendationIntroQuery(IGetSubTopicRecommendation getSubTopicRecommendationFromContentfulQuery) : IGetRecommendationIntro
+{
+    private readonly IGetSubTopicRecommendation _getSubTopicRecommendationFromContentfulQuery = getSubTopicRecommendationFromContentfulQuery;
+
+    public async Task<RecommendationIntro> GetRecommendationIntroForSubtopic(Section subTopic, Maturity maturity, CancellationToken cancellationToken = default)
```

Comment:
1. Is this query needed? I'm unsure of its purpose/use case.
2. Would it not be better to query the intro from Contentful directly, instead of querying the recommendation for the subtopic etc.? [^1]

[^1] Alternatively, assumedly this is called on the recommendation page somewhere, at which point we probably already have the data somewhere. Could we not just use that?

Reply:
After finishing rest of review, I do think that this Contentful query itself isn't needed, and this logic (i.e. finding the right intro for the subtopic) could be placed on the `SubtopicRecommendation` class.